setup.py

```diff
@@ -23,6 +23,7 @@ def read(fname):
         'kubernetes>=1.0.0,<2',
         'PyYAML>=3.12,<4',
         'jsonschema>=2.6.0,<3',
+        'boto3>=1.5.15'
```

Comment:
Could you please future-proof the `boto` version by setting it to `'boto3>=1.5.15,<2.0.0'` ? That would be awesome !